libimgdoc2/CMakeLists.txt

```diff
@@ -104,4 +154,7 @@ target_compile_definitions(libimgdoc2 PUBLIC _SQLITEIMGDOCSTATICLIB=1)
 
 target_include_directories(libimgdoc2 INTERFACE
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc> 
-  )
\ No newline at end of file
+  )
+
+  # add the binary tree to the search path for include files so that we will find libimgdoc2_config.h
```

Comment:
leading whitespace difference in comment and line it refers to

Reply:
fixed, thanks